src/scripts/components/foundations/buttons/IconButton.js

```diff
@@ -0,0 +1,26 @@
+/* @flow */
```

Comment:
@eliawk @jellegerbrandy @jrgarou I think we should use this for all of our icon buttons through the app. In fact I/we should probably also create a generic `Icon.js` component that this button component uses and would be useful itself in other contexts.